ui/src/store/store.ts

```diff
@@ -71,9 +50,9 @@ export const appStore = configureStore({
     }),
     middleware: (getDefaultMiddleware) => [
         ...getDefaultMiddleware({
-            // For now we don't need to store our redux store and therefore
-            // we don't need to ensure that everything in the store is serializable.
-            serializableCheck: false,
+            serializableCheck: {
+                ignoredActions: [FLUSH, REHYDRATE, PAUSE, PERSIST, PURGE, REGISTER],
```

Comment:
En ole täysin varma mihin kommentti viittasi. Redux-toolkitin doku ehdottaa redux-persistin yhteiskäytön määritettävän näin. Peräisin siis suoraan dokusta https://redux-toolkit.js.org/usage/usage-guide#use-with-redux-persist

Reply:
Tuo alkuperäinen kommentti on kyllä mielenkiintoinen, mullakaan ei ole oikeastaan hajuakaan mihin "we don't need to store our redux store" viittasi (yo dawg, I heard you like stores so we stored a store in your store ja sitä rataa) 🤔 Sinällään tosin meillä on ainakin ollut puhetta siitä, että ihan kaikkia reduxissa olevia avaimia ei aina tarvisi persistoida (reduxia käytetään monessa tilanteessa vaan kätevänä tapana frontin sisäiseen tilan passaamiseen komponentilta toiselle UI-puussa, ja tällaisen datan käyminen localStoragen kautta voi olla ei-haluttua ja jopa haitallista toiminnan kannalta.) En tosin muista onko tuollaisia memory-only-avaimia oikeasti toteutettuna vielä millekään, joskin oma mututuntumani sanoo että ei.

Reply:
Olisikohan tuo kommentti ehkä ajalta ennen kun redux storea tallennettiin localstorageen. Eli oppikirjaesimerkki kommenttien ikääntymisestä.